styles/layers.h

```diff
@@ -4,21 +4,28 @@
 #include "alpha.h"
 #include "../functions/int.h"
 #include "colors.h"
+#include "../common/color.h"
+#include <utility>
 
 // Usage: Layers<BASE, LAYER1, LAYER2, ...>
 // BASE: COLOR or LAYER
 // LAYER1, LAYER2: LAYER
 // return value: COLOR or LAYER (same as BASE)
-// This style works like layers in gimp or photoshop.
-// In most cases, the layers are expected to be normally transparent effects
-// that turn opaque when then want to paint an effect over the base color.
-// If the base color is opqaque, the final result of this style will also be
-// opaque. If the base color is transparent, the final result may also be transparent,
-// depending on what the layers paint on top of the base color.
+// This style works like layers in Photoshop: each layer is painted over the one beneath it.
```

Comment:
"beneath it" can be confusing, does it refer to the order of rending or the order of writing?
Not sure what a better way to describe it would be though...